### PR TITLE
Harvesting UI

### PR DESCRIPTION
**Pull request by OlegMakarenko**
image
image
image
image

### CODE REVIEW

File: src/language/en-US.json [hunk 1]
```diff
@@ -1052,6 +1058,10 @@
     "please_link_your_public_key": "Please relink your public key to generate your private key",
     "linking": "Linking...",
     "stoping": "Stopping...",
+    "request_harvesting": "Reqest Harvesting",
```
Comment (segfaultxavi) [inline src/language/en-US.json:1061]:
```suggestion
    "request_harvesting": "Request Harvesting",
```

File: src/language/en-US.json [hunk 2]
```diff
@@ -190,6 +190,12 @@
...
+    "harvesting_node_selection": "Please select the node from the dropdown list below. You can also find more nodes in the Symbol Explorer and then provide it by entering the URL.",
+    "open_explorer_node_list": "Visit Symbol Explorer the all node list",
+    "harvesting_keys_linked_next_step_guide": "Keys are linked. Now you can request a node to harvest for your account. If you want to change the node, you have to unlink the keys. Use buttons below",
+    "harvesting_delegated_request_warning": "There is possibility that the node will not accept your Delegated Harvesting reqest. It depends on account importance score and wether the node has a free slots.\n\nAlso please note that the node may remove any account from their harvester slots.\n\nIn these cases you will need to change the node and repeat the Delegated Harvesting activation process. Transaction fees paid for the activation will not be refunded.\n\nIf you entered a custom node URL, the Harvesting status may not be visible on other devices.",
+    "harvesting_delegated_request_warning_title": "Delegated Harvesting",
```
Comment (segfaultxavi) [inline src/language/en-US.json:198]:
```suggestion
    "harvesting_delegated_description": "Delegated Harvesting allows receiving rewards from creating new blocks using the account's importance score without running a node. To activate Delegated Harvesting you have to link all keys and request a node to harvest for your account.",
    "harvesting_node_selection": "Please select a node from the dropdown list below. You can find more nodes in the Symbol Explorer and use them by entering their URL.",
    "open_explorer_node_list": "Visit Symbol Explorer's node list",
    "harvesting_keys_linked_next_step_guide": "Keys are linked. You can now request a node to harvest for your account. If you want to change the node, you will have to unlink the keys first. Use buttons below.",
    "harvesting_delegated_request_warning": "Depending on your account's importance score and the node's free harvesting slots, your request might not be granted.\n\nAlso, please note that nodes may remove accounts from their harvesting slots at any time.\n\nIn these cases you need to switch to a different node and repeat the Delegated Harvesting activation process. Transaction fees paid for the activation will not be refunded.\n\nIf you entered a custom node URL, the Harvesting status shown may not be accurate on all your devices.",
    "harvesting_delegated_request_warning_title": "Delegated Harvesting",
```

**Review by segfaultxavi — CHANGES_REQUESTED** [review]

**Comment by OlegMakarenko** [comment]
@segfaultxavi Thank you!

**Comment by segfaultxavi** [comment]
image

**Pull request merged**